hydra_base/lib/storage/readers.py

```diff
@@ -401,5 +409,5 @@ def column_to_block_coord(column, blocks):
 def make_pandas_dataframe(index, series):
     return pd.DataFrame(
         {name: values for name, values in series.items()},
-        index=pd.DatetimeIndex(index)
+        index=pd.Index(index)
```

Comment:
If the incoming index is a DatetimeIndex this still works - the incoming index argument is a sequence of strings and the new index becomes an index of `object` types because no date parsing is applied. This also works for the string sequences that come from PeriodIndex, TimedeltaIndex etc.